docs/api/transport.md

```diff
@@ -57,6 +58,11 @@ Highlights on transporting functions are:
 - Closure cannot be transported, but you won't get error when transporting a function. Instead, you will get runtime error complaining a variable (from closure) is undefined when you can the function later.
 - `__dirname` / `__filename` can be accessed in transported function, which is determined by `origin` property of function. By default `origin` property is set to current working directory.
 
+### <a name="transporting-built-in"></a> Transporting JavaScript built-in object
+JavaScript standard built-in objects in [_builtInTypeWhitelist](./../../lib/transport/transport.ts) can be transported among napa workers transparently. JavaScript Objects with properties in these types are also able to be transported. Please refer to [transportBuiltinObjects](./../../test/transport-test.ts) for detail.
+
+In [Parallel Quick Sort](./../../examples/tutorial/parallel-quick-sort), a TypedArray based on a SharedArrayBuffer is transported among napa workers to perform quick sort in parallel.
```

Comment:
An example [Parallel Quick Sort] demonstrated transporting SharedArrayBuffer among multiple Napa workers for efficient data sharing.